### PR TITLE
Fixable flaws on the color_value page

### PR DESCRIPTION
**Pull request by rachelandrew**
Just a bunch of flaws on color_value that I spotted while working on another issue.

### CODE REVIEW

File: files/en-us/web/css/color_value/index.html [hunk 1]
```diff
@@ -1141,11 +1141,11 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
  <dt>-moz-mac-accentregularshadow</dt>
  <dt>-moz-mac-chrome-active</dt>
  <dd>
- <p>{{Gecko_minversion_inline("1.9.1")}}</p>
+ <p></p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1144]:
```suggestion
```

File: files/en-us/web/css/color_value/index.html [hunk 2]
```diff
@@ -1141,11 +1141,11 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
...
  </dd>
  <dt>-moz-mac-chrome-inactive</dt>
  <dd>
- <p>{{Gecko_minversion_inline("1.9.1")}}</p>
+ <p></p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1148]:
```suggestion
```

File: files/en-us/web/css/color_value/index.html [hunk 3]
```diff
@@ -1105,11 +1105,11 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
  </dd>
  <dt>-moz-Combobox</dt>
  <dd>
- <p>{{Gecko_minversion_inline("1.9.2")}} Background color for combo-boxes. Should be used with the <code>-moz-ComboboxText</code> foreground color. In versions prior to 1.9.2, use <code>-moz-Field</code> instead.</p>
+ <p> Background color for combo-boxes. Should be used with the <code>-moz-ComboboxText</code> foreground color. In versions prior to 1.9.2, use <code>-moz-Field</code> instead.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1108]:
```suggestion
 <p>Background color for combo-boxes. Should be used with the <code>-moz-ComboboxText</code> foreground color. In versions prior to 1.9.2, use <code>-moz-Field</code> instead.</p>
```

File: files/en-us/web/css/color_value/index.html [hunk 4]
```diff
@@ -1105,11 +1105,11 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
...
  </dd>
  <dt>-moz-ComboboxText</dt>
  <dd>
- <p>{{Gecko_minversion_inline("1.9.2")}} Text color for combo-boxes. Should be used with the <code>-moz-Combobox</code> background color. In versions prior to 1.9.2, use <code>-moz-FieldText</code> instead.</p>
+ <p> Text color for combo-boxes. Should be used with the <code>-moz-Combobox</code> background color. In versions prior to 1.9.2, use <code>-moz-FieldText</code> instead.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1112]:
```suggestion
 <p>Text color for combo-boxes. Should be used with the <code>-moz-Combobox</code> background color. In versions prior to 1.9.2, use <code>-moz-FieldText</code> instead.</p>
```

File: files/en-us/web/css/color_value/index.html [hunk 5]
```diff
@@ -1122,15 +1122,15 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
  <dt>-moz-dragtargetzone</dt>
  <dt>-moz-EvenTreeRow</dt>
  <dd>
- <p>{{gecko_minversion_inline("1.9")}} Background color for even-numbered rows in a tree. Should be used with the <code>-moz-FieldText</code> foreground color. In Gecko versions prior to 1.9, use <code>-moz-Field</code>. See also <code>-moz-OddTreeRow</code>.</p>
+ <p> Background color for even-numbered rows in a tree. Should be used with the <code>-moz-FieldText</code> foreground color. In Gecko versions prior to 1.9, use <code>-moz-Field</code>. See also <code>-moz-OddTreeRow</code>.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1125]:
```suggestion
 <p>Background color for even-numbered rows in a tree. Should be used with the <code>-moz-FieldText</code> foreground color. In Gecko versions prior to 1.9, use <code>-moz-Field</code>. See also <code>-moz-OddTreeRow</code>.</p>
```

File: files/en-us/web/css/color_value/index.html [hunk 6]
```diff
@@ -1122,15 +1122,15 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
...
  </dd>
  <dt>-moz-html-CellHighlight</dt>
  <dd>
- <p>{{gecko_minversion_inline("1.9")}} Background color for highlighted item in HTML {{HTMLElement("select")}}s. Should be used with the <code>-moz-html-CellHighlightText</code> foreground color. Prior to Gecko 1.9, use <code>-moz-CellHighlight</code>.</p>
+ <p> Background color for highlighted item in HTML {{HTMLElement("select")}}s. Should be used with the <code>-moz-html-CellHighlightText</code> foreground color. Prior to Gecko 1.9, use <code>-moz-CellHighlight</code>.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1129]:
```suggestion
 <p>Background color for highlighted item in HTML {{HTMLElement("select")}}s. Should be used with the <code>-moz-html-CellHighlightText</code> foreground color. Prior to Gecko 1.9, use <code>-moz-CellHighlight</code>.</p>
```

File: files/en-us/web/css/color_value/index.html [hunk 7]
```diff
@@ -1122,15 +1122,15 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
...
  </dd>
  <dt>-moz-html-CellHighlightText</dt>
  <dd>
- <p>{{gecko_minversion_inline("1.9")}} Text color for highlighted items in HTML {{HTMLElement("select")}}s. Should be used with the <code>-moz-html-CellHighlight</code> background color. Prior to Gecko 1.9, use <code>-moz-CellHighlightText</code>.</p>
+ <p> Text color for highlighted items in HTML {{HTMLElement("select")}}s. Should be used with the <code>-moz-html-CellHighlight</code> background color. Prior to Gecko 1.9, use <code>-moz-CellHighlightText</code>.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1133]:
```suggestion
 <p>Text color for highlighted items in HTML {{HTMLElement("select")}}s. Should be used with the <code>-moz-html-CellHighlight</code> background color. Prior to Gecko 1.9, use <code>-moz-CellHighlightText</code>.</p>
```

File: files/en-us/web/css/color_value/index.html [hunk 8]
```diff
@@ -1161,36 +1161,36 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
  </dd>
  <dt>-moz-MenuBarText</dt>
  <dd>
- <p>{{Gecko_minversion_inline("1.9.2")}} Text color in menu bars. Often similar to <code>MenuText</code>. Should be used on top of <code>Menu</code> background.</p>
+ <p> Text color in menu bars. Often similar to <code>MenuText</code>. Should be used on top of <code>Menu</code> background.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1164]:
```suggestion
 <p>Text color in menu bars. Often similar to <code>MenuText</code>. Should be used on top of <code>Menu</code> background.</p>
```

File: files/en-us/web/css/color_value/index.html [hunk 9]
```diff
@@ -1161,36 +1161,36 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
...
  </dd>
  <dt>-moz-MenuBarHoverText</dt>
  <dd>
  <p>Color for hovered text in menu bars. Often similar to <code>-moz-MenuHoverText</code>. Should be used on top of <code>-moz-MenuHover</code> background.</p>
  </dd>
  <dt>-moz-nativehyperlinktext</dt>
  <dd>
- <p>{{Gecko_minversion_inline("1.9.1")}} Default platform hyperlink color.</p>
+ <p> Default platform hyperlink color.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1172]:
```suggestion
 <p>Default platform hyperlink color.</p>
```

File: files/en-us/web/css/color_value/index.html [hunk 10]
```diff
@@ -1161,36 +1161,36 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
...
  </dd>
  <dt>-moz-OddTreeRow</dt>
  <dd>
- <p>{{gecko_minversion_inline("1.9")}} Background color for odd-numbered rows in a tree. Should be used with the <code>-moz-FieldText</code> foreground color. In Gecko versions prior to 1.9, use <code>-moz-Field</code>. See also <code>-moz-EvenTreeRow</code>.</p>
+ <p> Background color for odd-numbered rows in a tree. Should be used with the <code>-moz-FieldText</code> foreground color. In Gecko versions prior to 1.9, use <code>-moz-Field</code>. See also <code>-moz-EvenTreeRow</code>.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1176]:
```suggestion
 <p>Background color for odd-numbered rows in a tree. Should be used with the <code>-moz-FieldText</code> foreground color. In Gecko versions prior to 1.9, use <code>-moz-Field</code>. See also <code>-moz-EvenTreeRow</code>.</p>
```

File: files/en-us/web/css/color_value/index.html [hunk 11]
```diff
@@ -1161,36 +1161,36 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
...
  </dd>
  <dt>-moz-win-mediatext</dt>
  <dd>
- <p>{{gecko_minversion_inline("1.9")}} Should be used for text in objects with <code>{{cssxref("-moz-appearance")}}: -moz-win-media-toolbox</code>.</p>
+ <p> Should be used for text in objects with <code>{{cssxref("appearance")}}: -moz-win-media-toolbox</code>.</p>
  </dd>
  <dt>-moz-win-accentcolor</dt>
  <dd>
- <p>{{gecko_minversion_inline("56")}}<br>
+ <p><br>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1188]:
```suggestion
```

File: files/en-us/web/css/color_value/index.html [hunk 12]
```diff
@@ -1161,36 +1161,36 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
...
   Used to access the Windows 10 custom accent color that you can set on the start menu, taskbar, title bars, etc.</p>
  </dd>
  <dt>-moz-win-accentcolortext</dt>
  <dd>
- <p>{{gecko_minversion_inline("56")}}<br>
+ <p><br>
   Used to access the color of text placed over the Windows 10 custom accent color in the start menu, taskbar, title bars, etc.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1194]:
```suggestion
 <p>Used to access the color of text placed over the Windows 10 custom accent color in the start menu, taskbar, title bars, etc.</p>
```

File: files/en-us/web/css/color_value/index.html [hunk 13]
```diff
@@ -1205,11 +1205,11 @@ <h3 id="Mozilla_Color_Preference_Extensions">Mozilla Color Preference Extensions
  </dd>
  <dt>-moz-default-background-color</dt>
  <dd>
- <p>{{Gecko_minversion_inline("5.0")}} User's preference for the document background color.</p>
+ <p> User's preference for the document background color.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1208]:
```suggestion
 <p>User's preference for the document background color.</p>
```

File: files/en-us/web/css/color_value/index.html [hunk 14]
```diff
@@ -1205,11 +1205,11 @@ <h3 id="Mozilla_Color_Preference_Extensions">Mozilla Color Preference Extensions
...
  </dd>
  <dt>-moz-default-color</dt>
  <dd>
- <p>{{Gecko_minversion_inline("5.0")}} User's preference for the text color.</p>
+ <p> User's preference for the text color.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/color_value/index.html:1212]:
```suggestion
 <p>User's preference for the text color.</p>
```

**Review by ericwbailey — CHANGES_REQUESTED** [review]
Just some nitpicks for formatting and empty elements.

**Comment by nschonni** [comment]
Think a bit of this is covered in https://github.com/mdn/content/pull/2834, but i can rebase that if this lands first

**Review by ericwbailey — APPROVED** [review]

**Comment by ericwbailey** [comment]
@rachelandrew, @nschonni: Approved, but not merged considering #2834.

**Comment by nschonni** [comment]
@ericwbailey this one can land. I think it covers more in the specific file than my other PR. I'll just rebase that one after

File: files/en-us/web/css/color_value/index.html [hunk 15]
```diff
@@ -1161,37 +1159,35 @@ <h3 id="Mozilla_System_Color_Extensions">Mozilla System Color Extensions</h3>
...
+ <p>Default platform hyperlink color.</p>
  </dd>
  <dt>-moz-OddTreeRow</dt>
  <dd>
- <p>{{gecko_minversion_inline("1.9")}} Background color for odd-numbered rows in a tree. Should be used with the <code>-moz-FieldText</code> foreground color. In Gecko versions prior to 1.9, use <code>-moz-Field</code>. See also <code>-moz-EvenTreeRow</code>.</p>
+ <p>Background color for odd-numbered rows in a tree. Should be used with the <code>-moz-FieldText</code> foreground color. In Gecko versions prior to 1.9, use <code>-moz-Field</code>. See also <code>-moz-EvenTreeRow</code>.</p>
  </dd>
  <dt>-moz-win-communicationstext</dt>
  <dd>
- <p>{{gecko_minversion_inline("1.9")}} Should be used for text in objects with <code>{{cssxref("-moz-appearance")}}: -moz-win-communications-toolbox;</code>.</p>
```
Comment (nschonni) [inline files/en-us/web/css/color_value/index.html:1180]: Actually, should this stay `-moz-appearance`?

**Pull request merged**